lib/features/scan_nfc/nfc_scan_screen.dart

```diff
@@ -18,9 +18,20 @@ import 'package:go_router/go_router.dart';
 
 import '../profiles/cubit/profiles_cubit.dart';
 
-class NFCScanPage extends StatelessWidget {
+class NFCScanPage extends StatefulWidget {
   const NFCScanPage({super.key});
 
+  @override
+  State<NFCScanPage> createState() => _NFCScanPageState();
+}
+
+class _NFCScanPageState extends State<NFCScanPage> {
+  @override
+  void initState() {
+    context.read<ScanNfcCubit>().readTag();
+    super.initState();
```

Comment:
In initState we should first call the super and then do other stuff.

From the docs (if you hover over the code):
> Implementations of this method should start with a call to the inherited method, as in super.initState().